bin/build-css

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
-./node_modules/node-sass/bin/node-sass $@ \
-    assets/sass/style.scss \
-    assets/stylesheets/application.css
+./node_modules/.bin/sass \
+    --no-source-map \
+    --quiet-deps \
```

Comment:
we get some deprecation warnings with dart-sass due to [this defect](https://github.com/alphagov/govuk-frontend/issues/2238).

The --quiet-deps flag drops it from many to 3 warnings